src/fidasim.f90

```diff
@@ -8997,50 +8999,50 @@ subroutine gyro_step(vi, fields, r_gyro)
         !+ Gyro-radius vector from particle position to guiding center
 
     real(Float64), dimension(3) :: vxB, rg_uvw, uvw, cuvrxb, b_rtz, grad_B, rg_rtz
-    real(Float64) :: one_over_omega, phi, R, rg_r, vpar, term1, term2
+    real(Float64) :: one_over_omega, phi, R, vpar, term1, term2
 
-    if(inputs%no_flr.eq.0) then
+    if(inputs%no_flr.lt.2) then
```

Comment:
@deyongl, @alvin-garcia What do you think about changing `inputs%no_flr` to `inputs%flr` where the number it is set to refers to the correction order? So `inputs%flr = 0` would mean no gyro correction, `inputs%flr = 1` would just include the the first order correction (`r_gyro = vxB*one_over_omega`) and `inputs%flr=2` would include the second order corrections